tests/stests/control_interface/authorization.robot

```diff
@@ -0,0 +1,143 @@
+*** Comments ***
+# Copyright (c) 2023 Elektrobit Automotive GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+*** Settings ***
+Documentation       Tests the authorization of the Control Interface access from workloads.
+
+Resource            ../../resources/ankaios.resource
+Resource            ../../resources/variables.resource
+
+*** Variables ***
+${simple_yaml_file}      ${EMPTY}
+${manifest12_yaml_file}  ${EMPTY}
+${manifest1_yaml_file}   ${EMPTY}
+${manifest2_yaml_file}   ${EMPTY}
+${manifest_no_agent_name_yaml_file}    ${EMPTY}
+
+*** Test Cases ***
+
+No rules
```

Comment:
As we mentioned BDT in our [contribution guidelines](https://eclipse-ankaios.github.io/ankaios/latest/development/system-tests/#behavior-driven-system-test), we should write the system tests in that way. But If we decide to not to be that strict, then we should mention this in the contribution guidelines!

Reply:
I have changed the stests to follow the BDT style.